MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,423 @@
+---
+layout: post
+title: Selection in .NET MAUI Segmented control | Syncfusion
+description: Learn about the Selection in Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfSegmentedControl
+documentation: ug
+---
+ 
+# Selection in .NET MAUI Segmented (SfSegmented) control
+
+This section explains about features that aid in the selection of items, customization of the selected item, and its related operations in the Segmented Control.
```

Comment:
This section describes the features of the Segmented Control that help with item selection, customization of the selected item, and associated operations.

Reply:
changed content